.env.example

```diff
@@ -10,6 +10,22 @@ CUBEJS_DB_TYPE=postgres
 CUBEJS_API_SECRET= # random 32-digit base64 string - generate from https://generate.plus/en/base64
 CUBEJS_DEV_MODE=true
 
+PORT=4000
+
```

Comment:
wouldn't we need more DB env variables like: `CUBEJS_DB_SCHEMA` (multi-tenancy), `CUBEJS_DB_SSL`? 
Are we using pre-aggregations? are we using Redshift or RDS?

Reply:
I think we can live without these keys for now.

The DB_SCHEMA doesn't have much documentation. Maybe it can help solving the issue with multitenancy that we're facing. Will look into this.